### PR TITLE
feature: add/remove testAdapter scripts add

### PR DESCRIPTION
**Pull request by jerrylib**
1. Add or remove testAdapter in Vault exchange array

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -56,6 +56,8 @@
     "coverage": "hardhat coverage",
     "docgen": "hardhat docgen",
     "topup": "hardhat run scripts/top-up.js --network localhost",
+    "adapter_add": "hardhat run scripts/test-adapter-add.js --network localhost",
```
Comment (mingshengke) [inline package.json]: rename to test_adapter_add and test_adapter_remove

File: scripts/test-adapter-remove.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
+const Vault = hre.artifacts.require("Vault");
+const ExchangeAggregator = hre.artifacts.require("ExchangeAggregator");
+
+// === Utils === //
+const { findIndex } = require("lodash");
+
+const VAULT_ADDRESS = "0x8a8fA8b26B8D2743ECd3c687cCDB345061D70540";
```
Comment (mingshengke) [inline scripts/test-adapter-remove.js:8]: Explain where the address is from

**Review by mingshengke — APPROVED** [review]

File: package.json (hunk 1 above)
Comment (jerrylib) [reply package.json]: Done

File: scripts/test-adapter-remove.js (hunk 2 above)
Comment (jerrylib) [reply scripts/test-adapter-remove.js:8]: Done

**Pull request merged**